hw2/ViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ViewController.swift
+//  hw2
+//
+//  Created by Tina Jureško on 14.03.2025..
+//
+
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class ViewController: UIViewController {
+
+    private let leagueView = LeagueView()
+    //private let matchView = MatchView()
+    
+    private let matchesStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        view.addSubview(leagueView)
+        leagueView.translatesAutoresizingMaskIntoConstraints = false
+        leagueView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(20)
+            make.leading.equalTo(view.safeAreaLayoutGuide.snp.leading).offset(16)
+            make.trailing.equalTo(view.safeAreaLayoutGuide.snp.trailing).offset(-16)
+            make.height.equalTo(56)
+        }
+        
+        view.addSubview(matchesStackView)
+        matchesStackView.snp.makeConstraints { make in
+            make.top.equalTo(leagueView.snp.bottom).offset(16)
+            make.leading.equalTo(view.safeAreaLayoutGuide.snp.leading).offset(16)
+            make.trailing.equalTo(view.safeAreaLayoutGuide.snp.trailing).offset(-16)
+        }
+
+        
+        let league = Homework2DataSource().laLigaLeague()
```

Comment:
Izbjegavaj nazive Homework2DataSource, pošto se svaka zadaća nastavlja na drugu pa trebaju biti univerzalni nazivi. Na PRu vidim što se sve rađeno pa znam koja je zadaća :)